### ISSUE TITLE
implement chromaticity in framework to improve code quality

### ISSUE THREAD

**Issue by velrest** [johnfrancisgit/chromaticity#13]
__Current Frameworks in list__
* Ember
* React
* Angular

__TODO__
* Check other frameworks to use with electron
* test frameworks for usability

**Comment by johnfrancisgit** [comment]
Another possibility would be to just use JS with handlebars as the templating engine. It would save us some time as we would have to re-write less code.
Code quality might be better with a consistent framework though.

**Comment by velrest** [comment]
@johnfrancisgit True. i will edit since ember and angular are probably overkill and take too much time.

**Comment by johnfrancisgit** [comment]
Should we kill this project?
